features/step_definitions/sign_up.rb

```diff
@@ -0,0 +1,80 @@
+require 'faker'
+
+# Navigation to sign up page:
+
+Given('I am on the login page') do
+  navigate_to(ENV['URL'])
+end
+
+And('I click on the registration button') do
+  click(:id, 'sign-up')
+end
+
+# Successful registration:
+
+When('I fill all the required fields') do
+  enter_text(:id, Faker::Name.first_name, 'firstname')
+  enter_text(:id, Faker::Name.last_name, 'lastname')
+  enter_text(:id, Faker::Internet.email(domain: 'wolox.com.ar'), 'email')
+  enter_text(:id, Faker::Internet.password, 'password')
+end
+
+And('I click on the submit button') do
+  click(:id, 'submit_register')
+end
+
+Then('I will be redirected to the login page') do
+  wait(2) # por qué??
```

Comment:
For the handling of waits I recommend to use the explicit Wait (it will execute a waiting time without fulfilling any condition), you can say that while x element is not visible it waits for a time 

#wait_for_element_to_display(access_type, access_name, duration) ⇒ Object
#wait_for_element_to_enable(access_type, access_name, duration) ⇒ Object